validation/views.py

```diff
@@ -484,9 +492,99 @@ def register(request):
     return render(request, "validation/register.html", context)
 
 
+def view_issues(request):
+    issues = Issue.objects.filter(status=Issue.OPEN).order_by("id")
+    context = dict(
+        issues=issues,
+        auth=request.user.is_authenticated,
+        is_linguist=user_is_linguist(request.user),
+    )
+    return render(request, "validation/view_issues.html", context)
+
+
+def view_issue_detail(request, issue_id):
+    issue = Issue.objects.filter(id=issue_id).first()
+
+    form = None
+    if issue.recording:
```

Comment:
If you want to make this a bit cleaner, I suggest making two new functions: one that handles `issue.recording` and one that handlers `issue.phrase`. It seems like the big `if`/`elif` do not share a lot of code, so you could totally split them up into separate functions without as many indents!